execute/table.go

```diff
@@ -721,10 +723,11 @@ type TableBuilder interface {
 	AppendFloats(j int, values []float64) error
 	AppendStrings(j int, values []string) error
 	AppendTimes(j int, values []Time) error
+
 	// TODO(adam): determine if there's a useful API for AppendValues
 	// AppendValues(j int, values []values.Value)
 
-	// GrowBools and similar functions will extend column j by n zero-values for the respective type.
+	// GrowBools and similar functions will extend column j by n nil-values for the respective type.
```

Comment:
I think it would probably be better to keep this function the same. That way, you aren't setting a bunch of values on a map that may not need to be set.